samples/display/ili9340/nrf52840_pca10056.overlay

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&spi1 {
+	status = "ok";
```

Comment:
In this sample, the Adafruit LCD shield 1947 is plugged in the Arduino-compatible connector on the nRF52840 DK (PCA10056). These GPIO pins are selected to match the hardware configuration.

For the 'standard' pins, the user has to connect the two boards by wiring.

Reply:
OK, so either the Nordic boards fail to specify a SPI driver that aligns to Arduino pins, which is not your problem, or the shields don't match what Nordic provided.  In either case, please document in the overlay that these specific pins were chosen to support Arduino shields, and maybe provide the readable names  (P1.15 instead of 47).

Also note that in this case RESET needs to be tied to Vdd since there isn't a gpios property for it.  Somebody could spend a lot of time trying to figure out what's wrong if that signal isn't connected on the Adafruit display.

Most of this documentation really belongs in the README, with just a reference to Arduino shield compatibility in the overlay.

Reply:
The README will be updated for the additional hardware configurations. The Zephyr documentation also needs to be updated. If possible, I will make it in another PR.

The Adafruit LCD shield 1947 works without a RESET pin driven by the ili9340 driver. Will give the GPIO details in README.